Index: Tehtava2.4.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Tehtava2.4.py b/Tehtava2.4.py
new file mode 100644
--- /dev/null	(date 1673340527252)
+++ b/Tehtava2.4.py	(date 1673340527252)
@@ -0,0 +1,9 @@
+luku1 = int(input("Syötä ensimmäinen luku "))
+luku2 = int(input("Syötä toinen luku"))
+luku3 = int(input("Syötä kolmas luku"))
+summa = luku1+luku2+luku3
+tulo = luku1*luku2*luku3
+keskiarvo = (luku1+luku2+luku3)/3
+print("Lukujen summa on " + str(summa))
+print("Lukujen tulo on " + str(tulo))
+print("Lukujen keskiarvo on " + str(keskiarvo))
\ No newline at end of file
Index: Tehtava2.5.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Tehtava2.5.py b/Tehtava2.5.py
new file mode 100644
--- /dev/null	(date 1673356721079)
+++ b/Tehtava2.5.py	(date 1673356721079)
@@ -0,0 +1,10 @@
+import math
+syotto1 = float(input("Syötä leiviskat"))
+syotto2 = float(input("Syötä naulat"))
+syotto3 = float(input("Syötä luodit"))
+naulat = (syotto1 * 20) + syotto2
+luodit = (naulat * 32) + syotto3
+massa = (luodit * 13.3)
+kilot = massa//1000
+grammat = massa%1000
+print("Massa nykymitoilla on: "f"Kilogrammat {kilot:2.0f}"+ f" ja Grammat {grammat:6.2f}")
Index: Tehtava2.6.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Tehtava2.6.py b/Tehtava2.6.py
new file mode 100644
--- /dev/null	(date 1673433479888)
+++ b/Tehtava2.6.py	(date 1673433479888)
@@ -0,0 +1,12 @@
+import random
+eka = random.randint(0,9)
+toka = random.randint(0,9)
+kolmas = random.randint(0,9)
+koodi = [eka,toka,kolmas]
+print("tässä on kolmenumeroinen koodisi " + str(koodi))
+first = random.randint(1,6)
+second = random.randint(1,6)
+third = random.randint(1,6)
+fourth = random.randint(1,6)
+code = (first,second,third,fourth)
+print("Tässä on nelinumeroinen koodisi " + str(code))
\ No newline at end of file
Index: Tehtava2.1.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Tehtava2.1.py b/Tehtava2.1.py
new file mode 100644
--- /dev/null	(date 1673336598081)
+++ b/Tehtava2.1.py	(date 1673336598081)
@@ -0,0 +1,3 @@
+print ("Mikä on nimesi?")
+nimi = input()
+print("Hei " + nimi + "!")
\ No newline at end of file
Index: Tehtava2.2.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Tehtava2.2.py b/Tehtava2.2.py
new file mode 100644
--- /dev/null	(date 1673337254302)
+++ b/Tehtava2.2.py	(date 1673337254302)
@@ -0,0 +1,7 @@
+import math
+print ("Syötä ympyrän säde")
+sade = input()
+sade = float(sade)
+x = math.pi
+ala = sade * x
+print(ala)
\ No newline at end of file
Index: Tehtava2.3.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Tehtava2.3.py b/Tehtava2.3.py
new file mode 100644
--- /dev/null	(date 1673338650741)
+++ b/Tehtava2.3.py	(date 1673338650741)
@@ -0,0 +1,7 @@
+print("Syötä kolmion kanta")
+kanta = input()
+print("Syötä kolmion korkeus")
+korkeus = input()
+ala = (int(kanta)) * (int(korkeus))
+piiri = (int(kanta))*2 + (int(korkeus))*2
+print("Kolmion pinta-ala on " + (str(ala))+" ja sen piiri on " + (str(piiri)))
\ No newline at end of file
