Index: Tehtava1.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print (\"Hei Roger Berg!\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Tehtava1.py b/Tehtava1.py
--- a/Tehtava1.py	(revision e775517eb62805a3b18fae6cc5d55de6189a2a1e)
+++ b/Tehtava1.py	(date 1673336321219)
@@ -1,1 +1,1 @@
-print ("Hei Roger Berg!")
\ No newline at end of file
+print ("Hei Roger Berg!")
